src/models/action.ts

```diff
@@ -16,12 +16,11 @@ import { ActionDescription as AddonActionDescription, Any } from 'gateway-addon/
 import Thing from './thing';
 
 export interface ActionDescription {
-  input: Any;
   href: string;
   status: string;
   timeRequested: string;
   timeCompleted?: string;
-  error?: string;
+  error?: Record<string, unknown>;
```

Comment:
Have you considered making a Problem Details interface? I'm reading from the profile spec that also other methods MAY explain error codes using that format. It might be nice to have such an interface handy in the future. (here is [zion's Problem Details](https://github.com/vaimee/zion/blob/main/src/common/interfaces/problem-details.ts) interface as a reference)

Reply:
Good idea, I've done that.